01-lets_get_started/01.rb

```diff
@@ -5,25 +5,26 @@ def initialize(side)
     @gap_length = side - 2
   end
 
-  def first_last_line
+  def print_first_last_line
     @side.times { print '#' }
```

Comment:
Do you want to use an enumerator for this? Can't you use a `string` & `integer` multiplication to generate the required vector?